1_slices_modules_and_objects/exercises/acronym/src/lib.rs

```diff
@@ -10,7 +10,7 @@
 /// ```
 // Side note: this doc example is also tested when running `cargo test`
 pub fn make_acronym(name: &str) -> String {
-    unimplemented!()
+    String::from(name).split_whitespace().map(|word| word.chars().nth(0).unwrap().to_ascii_uppercase()).collect::<String>()
```

Comment:
Good! You can also split a `&str` so you don't have to create a new string each time, for example using `name.split(" ")`